src/app/pages/psbt-request/components/psbt-decoded-request/psbt-decoded-request-views/build-psbt-input-for-ui.ts

```diff
@@ -0,0 +1,46 @@
+import { BitcoinNetworkModes } from '@shared/constants';
+import { getAddressFromOutScript } from '@shared/crypto/bitcoin/bitcoin.utils';
+import { isDefined } from '@shared/utils';
+
+import { OrdApiInscriptionTxOutput } from '@app/query/bitcoin/ordinals/ordinals-aware-utxo.query';
+
+import { PsbtInput, PsbtInputForUi } from './use-psbt-decoded-request';
+
+function getPsbtInputValue(input: PsbtInput, unsignedUtxo?: OrdApiInscriptionTxOutput) {
```

Comment:
How come the latter is optional?

Reply:
Bc Gamma (and others) often send 'dummy' unsigned inputs in the psbt, so the ord api doesn't return data.